Constants.java

```diff
@@ -17,11 +17,19 @@ private Constants() {}
       0x1, 0x2, 0x4, 0x8, 0x10, 0x20, 0x40, 0x80, 0x100
     };
 
+    //Boards that have two in a rows
+    public static final int[] TWOINAROWS = {0x180, 0xC0, 0x30, 0x18, 0x6, 0x3, 0x120, 0x24, 0x90, 0x12, 0x48, 0x9, 0x110, 0x11, 0x50, 0x14};
+    
+    //The elements in OPENTHIRDS correspond to the third squares that have to be open for each element of TWOINAROWS to have a chance of getting a win
+    public static final int[] OPENTHIRDS = {0x40, 0x100, 0x8, 0x20, 0x1, 0x4, 0x4, 0x100, 0x2, 0x80, 0x1, 0x40, 0x1, 0x100, 0x4, 0x40};
+
     // ASCII codes for printing boards
     public static final int O_ASCII = 79;
     public static final int X_ASCII = 88;
 
     // programming constants for debugging, etc.
     public static final int REPORTING_LEVEL = 3;
     public static final int PRINT_DETAIL = 1;
+
```

Comment:
Some note about which constant means what would be nice. i.e, EVAL_CONSTANTS[0] is for blah blah, 1 is for blah blah